galasa-managers-parent/galasa-managers-testingtools-parent/dev.galasa.selenium.manager.ivt/src/main/java/dev/galasa/selenium/manager/ivt/SeleniumManagerIVT.java

```diff
@@ -20,24 +25,29 @@
     @SeleniumManager
     public ISeleniumManager seleniumManager;
 
+    public final String WEBSITE = "https://duckduckgo.com";
```

Comment:
must be static,  same for the following 3,   else SQ will moan about the variable name